news/2024/2024-07-01-the-followpoint-neuro-sama-the-ai-vtuber-that-invaded-osu.md

```diff
@@ -0,0 +1,118 @@
+
+*How did you come up with your username and is there any special meaning behind it?*
+
+> There is no special meaning behind it, and honestly, I don't think it's a great username. And it can also be confusing, as when Neuro is streaming it will always be coming from my channel, vedal987, so some people - especially at the start - thought that Neuro's name was Vedal.
+
+*How did you get introduced to osu! and what was your first impression of it?*
+
+> I first played osu! in early 2017, and though I can't remember exactly why I suspect I might have found the game through [BTMC](https://osu.ppy.sh/users/3171691)'s "And some say he never played osu! again" video. osu! was my first experience with a rhythm game, so it was nice to have something completely different - especially when it's free. I'd play with a few friends every now and then, which was quite fun as I'm a pretty competitive person, making improving that much more satisfying.
+
+*What would you call the greatest gift that osu! has given you?*
+
+> Maybe second to carpal tunnel, I'd say it's not an exaggeration to say it was the springboard for my career and the situation I'm in now, so I'm appreciative to the game and the community for embracing Neuro as much as they did - without them I definitely wouldn't be here like this today. So thank you osu!
+
+*Would **you** still be playing osu! if you had time for it?*
+
+> I think for sure I'd play it every now and then, I was never cut out for playing it competitively though. Right now, I have a lot of things I'm cooking up and things I'm looking forward to with Neuro, but maybe I'll be back to osu! one day.
+
+*What other rhythm games besides osu! have you played?*
+
+> I personally haven't tried too many other rhythm games, although I've been known to enjoy some Beat Saber from time to time. I've honestly not really played many games in general recently, with the exception of games on stream.
+
+*How did you come up with OG Neuro (AI learning to play osu!) idea and why exactly in osu! of all games? Also I would like to know what made you decide to revive Neuro as a VTuber in 2022?*
+
+> I was just getting started with learning AI in 2019, and I happened to be playing a lot of osu! at the time. I'd always found it super interesting and cool when people made AI's to play games, and I had an idea for how to do it with osu!. The original AI took a few days to make and I thought it would be a cool idea to stream it - to track its progress. Eventually after improving the AI, its performance stagnated and I wasn't sure what to do with it since I had no idea what I was doing at the time. Two years later, I had the idea to make an AI VTuber although it wasn't until early 2022 that I had the idea to combine the original Neuro-sama with this concept.
```

Comment:
felt like the pause was not needed there, subjective though
```suggestion
> I was just getting started with learning AI in 2019, and I happened to be playing a lot of osu! at the time. I'd always found it super interesting and cool when people made AI's to play games, and I had an idea for how to do it with osu!. The original AI took a few days to make, and I thought it would be a cool idea to stream it in order to track its progress. Eventually after improving the AI, its performance stagnated, and I wasn't sure what to do with it since I had no idea what I was doing at the time. Two years later, I had the idea to make an AI VTuber, although it wasn't until early 2022 that I had the idea to combine the original Neuro-sama with this concept.
```